.github/workflows/ci.yml

```diff
@@ -4,7 +4,7 @@ on: [push]
 
 env:
   evomaster-version: 1.2.2
-  release-jdk: 17
+  release-jdk: 16
```

Comment:
looks like you had problem with the merging of master... and you overwrite incoming changes :( all those have to be reverted

Reply:
Hi, I revert them two times and it starts to loose some portion of code. So I create a new branch, all the problems are resolved and it is more clear since the comments in this branch belongs to another branch.